WordPress/WordPressNotificationServiceExtension/Sources/NotificationService.swift

```diff
@@ -71,6 +71,10 @@ class NotificationService: UNNotificationServiceExtension {
                     notificationContent.body = notificationText
                 }
 
+                // NB: placeholder pending `note_type`
+                let placeholderNoteType = "replyto-like-comment"
+                notificationContent.categoryIdentifier = placeholderNoteType
```

Comment:
I don't think you should need to override this – I tested and the `notificationContent`'s `categoryIdentifier` was already `replyto-like-comment` at this point (it's set on the server side).